docs/docusaurus.config.js

```diff
@@ -0,0 +1,92 @@
+const remarkSources = require('remark-sources');
+const remarkCollapse = require('remark-collapse'); // TODO: probably won't need this one
+const { remarkProgramOutput } = require("./plugins/program_output");
+
+let versions = [];
+try {
+  versions = require('./versions.json');
+} catch (ex) {
+  // Nothing to do here, in dev mode, only
+  // one version of the doc is available
+}
+let legacyVersions = [];
+
+module.exports = {
+  title: 'Rasa SDK Documentation',
+  tagline: 'Rasa SDK Documentation',
+  // TODO: is it needed?
+  url: 'http://something.com',
+  baseUrl: '/docs/rasa/rasa-sdk/',  //TODO: seems fake
+  onBrokenLinks: 'throw',
+  favicon: 'img/favicon.ico',
```

Comment:
can you use the same favicon as for rasa ?

Reply:
i copied that from here: https://github.com/RasaHQ/rasa/blob/master/docs/docusaurus.config.js#L25

should both of them be the one in L29, you mean?

Reply:
yes, but the file is not the right one (it's the docusaurus favicon). I couldn't comment on the image file, so I commented here

Reply:
ohhhh i see!